config/locales/en.yml

```diff
@@ -1,4 +1,7 @@
 en:
+  time:
+    formats:
+      short: "%d %b %Y"
```

Comment:
Do you also need to do this for Date objects? Can be a bit of a gotcha. See http://nicholshayes.co.uk/blog/?p=223